extensions/terminal-suggest/src/shell/zshBuiltinsCache.json

```diff
@@ -1,371 +1,499 @@
 {
-  "-": {
-    "description": "See the section `Precommand Modifiers' in zshmisc(1).",
-    "args": "- simple command"
-  },
-  ".": {
-    "shortDescription": "Source a file",
-    "description": "Read commands from file and execute them in the current shell environment. If file does not contain a slash, or if PATH_DIRS is set, the shell looks in the components of $path to find the directory containing file.  Files in the current directory are not read unless `.' appears somewhere in $path.  If a file named `file.zwc' is found, is newer than file, and is the compiled form (created with the zcompile builtin) of file, then commands are read from that file instead of file. If any arguments arg are given, they become the positional parameters; the old positional parameters are restored when the file is done executing.  However, if no arguments are given, the positional parameters remain those of the calling context, and no restoring is done. If file was not found the return status is 127; if file was found but contained a syntax error the return status is 126; else the return status is the exit status of the last command executed.",
-    "args": ". file [ arg ... ]"
-  },
-  ":": {
-    "description": "This command does nothing, although normal argument expansions is performed which may have effects on shell parameters.  A zero exit status is returned.",
```

Comment:
missing some, like alias

Reply:
Will look into this before merging, this actually adds a bunch but apparently removed some too. Probably just need to tweak the regexes